### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by rsedlock1958**
# Description

Minor edits to comply with Dell style guidelines
# Issues

List the issues impacted by this PR:

| Issue ID |
| -------- |
|   https://github.com/dell/karavi-observability/issues/28       |

# Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have run 'make check' to ensure my code does not have any formatting, vetting, linting, or security issues
- [ ] I have run 'make test' to ensure new and existing unit tests pass locally with my changes
- [ ] I have not allowed coverage numbers to degrade
- [ ] I have maintained at least 90% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

### CODE REVIEW

**Comment by ioeicicdsvc** [comment]
@rsedlock1958: Thank you for submitting your pull request!

Please make sure you have followed our contributing guidelines. We will review it as soon as possible.

------------------------------------------------------------------------------------------------------------------------
We want your feedback! If you have any questions or suggestions regarding our contributing process/workflow, please reach out to us at karavi@dell.com.

**Comment by ameer-jabbar** [comment]
Can one of the admins verify this patch?

**Review by AronAtDell — APPROVED** [review]
Look good.

File: internal/service/configuration_reader.go [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+	config, err := ioutil.ReadFile(filepath.Clean(file))
+	if err != nil {
+		return nil, fmt.Errorf(fmt.Sprintf("File %s errors: %v", file, err))
+	}
+
+	var storageSystem ArrayConnectionData
+
+	if string(config) == "" {
+		return nil, fmt.Errorf("arrays details are not provided in vxflexos-config secret")
```
Comment (alexemc) [inline internal/service/configuration_reader.go:47]: The first letter case in the error messages is inconsistent. Shouldn't all messages start from a lowercase letter?

**Comment by tdawe** [comment]
For some reason, this branch contains a commit that is in another branch for supporting changes in the CSI driver. You can create a new branch off of main and add the single commit with documentation changes.

**Comment by gallacher** [comment]
Closing PR because it contains an invalid commit

**Pull request closed without merging**